WalletWasabi.Gui/Tabs/WalletManager/LoadWalletViewModel.cs

```diff
@@ -569,9 +570,8 @@ public async Task LoadWalletAsync()
 			}
 			finally
 			{
-				IsBusy = false;
 				MainWindowViewModel.Instance.StatusBar.RemoveStatus(loadingStatusText);
```

Comment:
Move this to the last line of try block above? I don't like the feeling to have an exception here. To much logic in RemoveStatus.